app/javascript/components/ingredients/ingredients-edit-inventory.vue

```diff
@@ -58,7 +57,8 @@
             class="flex flex-col w-full 2xl:justify-center items-center overflow-auto"
           >
             <inventory-table
-              :ingredients="filterIngredients"
+              :ingredients="ingredients"
+              :searchQuery="searchQuery"
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Attribute ':searchQuery' must be hyphenated  vue/attribute-hyphenation